File: kiki616/ABC/B/318.py
N = int(input())
table = [[0 for l in range(100)] for i in range(100)]
cnt = 0
for i in range(N):
    a,b,c,d = map(int,input().split())
    for l in range(a,b):
        for k in range(c,d):
            table[l][k] = 1
for i in table:
    for l in i:
        if l == 1:
            cnt = cnt +1
print(cnt)

File: OCH_Tao/ABC/A/318.js
const [N, M, P] = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ").map(x => parseInt(x, 10));
let tmp = N - M;
let cnt = 0;
while (tmp >= 0) {
  cnt++;
  tmp -= P;
}
console.log(cnt);